code/game/machinery/teleporter.dm

```diff
@@ -209,7 +216,7 @@
 			O.show_message("<span class='warning'>Failure: Cannot authenticate locked on coordinates. Please reinstate coordinate matrix.</span>")
 		return
 	if(istype(M, /atom/movable))
-		if(prob(5) && !accurate) //oh dear a problem, put em in deep space
+		if(prob(failprob) && !accurate) //oh dear a problem, put em in deep space- and if someone's tampered with the teleporter, even testfires won't help
```

Comment:
This should probably have a check for if `testfire_broken` is true or false.
I.E. `(prob(failprob) && (!accurate || testfire_broken))`